javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,117 @@
+let name = ""
+const todoList = []
+
+function getReply(command) {
+ if (typeof command !== "string" || command === "") {
+   return "Please enter a valid command"
+ }
+/* let num1 = x;
+let num2 = y;*/
+ switch(command){
+
+   case "Hello my name is Benjamin":
+     name = "Benjamin"
+       return `Nice to meet you Benjamin`
+
+   case "What is my name?":
+       if(name !== ""){
+         return "You didn't say your name yet"
+       }
+   case "Add Fishing to my todos":
+        if (command !== "") {
+        todoList.push("Fishing")
+        return "Added Fishing to your todos"
+   }
+   
+   case "Add Singing in the shower to my todos":
+        if (command !== "") {
+        todoList.push("Singing")
+        return "Added Singing in the shower to your todos"
+    }
+
+    case "What on my todos":
+    if (command !== "") {
```

Comment:
You should check if the command includes `What on my todos` and then return the sentence `You have ${todoList.length} todos - ${todoList};`